servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -207,6 +207,13 @@ def get_task_attempts(id, task, **kwargs):
     raise NotImplementedError('Tasks not supported by dsub.')
 
 
+def get_operation_details():
```

Comment:
I have a mild preference for stub functions to explode rather than do nothing when called